swagger.yaml

```diff
@@ -7013,8 +7035,55 @@ definitions:
         description: Indication that the version is the recommended one.
       support_level:
         type: string
-        enum: [beta, production, maintenance]
+        enum: [beta, production, maintenance, end-of-life]
         description: Level of support of the version.
+  
+  release-source:
+    type: object
+    required:
+      - openshift_version
+      - multi_cpu_architectures
+      - upgrade_channels
+    properties:
+      openshift_version:
+        type: string
+        description: Version of the OpenShift cluster.
+        example: "4.14"
+      multi_cpu_architectures:
+        type: array
+        items:
+          type: string
+          enum: ['x86_64', 'aarch64', 'arm64','ppc64le','s390x','multi']
+          description: Supported CPU architecture for multi-architecture releases in this OpenShift version..
+      upgrade_channels:
+        type: array
+        items:
+          $ref: '#/definitions/upgrade-channel'
+
+  upgrade-channel:
+    type: object
+    required:
+      - cpu_architecture
+      - channels
+    properties:
+      cpu_architecture:
+        type: string
+        enum: ['x86_64', 'aarch64', 'arm64','ppc64le','s390x','multi']
```

Comment:
Same here I don't think we should have both `arm64` and `aarch64` also is `multi` actually valid for a channel?

Reply:
Same as for `arm64/aarch64` as last comment, but here multi is a valid release to pick from the API